Gemfile

```diff
@@ -74,6 +74,8 @@ group :development do
 
   # opens emails for us
   gem 'letter_opener'
+
+  gem 'rails-erd'
```

Comment:
How about a comment to explain what the gem does for us. Helps when one day we are attempting to clean out gems we don't use anymore

Reply:
@lamorrison I have to install GraphViz, can we mention this here too?